scss/civicrm/grant/_grant.scss

```diff
@@ -48,6 +48,10 @@
         td {
           font-size: $font-size-base !important;
           padding: 15px 20px !important;
+
+          &:last-child {
```

Comment:
Cant we use a class/attribute/something else to identify the `td`? 
What if there is another td added. this will break in the case.

What is the reason of the field being right aligned in the first place.

Reply:
We have common tables everywhere having last column to be on right aligned always but, in some cases we make it left. so, it's the right one. 

Also, the structure is core so, we don't change it until breaking or wrong something in there. we just fix with css.